data/tlg2001/tlg043/tlg2001.tlg043.opp-lat1.xml

```diff
@@ -90,16 +90,16 @@
     </encodingDesc>
  <profileDesc>
   <langUsage>
-  <language ident="grc">Greek</language>
   <language ident="lat">Latin</language>
   </langUsage>
  </profileDesc>
  <revisionDesc>
+   <change who="Brittany Hardy" when="2017-06-12"> changed file name from grc to lat, corrected lang tags and cts metadata file accordingly, could not find scans online to add source </change>
   <change/>
  </revisionDesc>
  </teiHeader>
   <text>
-    <body><div type="edition" n="urn:cts:greekLit:tlg2001.tlg043.opp-grc1" xml:lang="grc">
+    <body><div type="edition" n="urn:cts:greekLit:tlg2001.tlg043.opp-lat1" xml:lang="lat">
```

Comment:
The `type` on this `<div>` element should be "translation" instead of "edition".